### DISCUSSION TITLE
[QUESTION] Neoforge support?

### DISCUSSION THREAD

**Discussion by championhestu** [P3pp3rF1y/SophisticatedBackpacks#964] [Q&A]
Will Sophisticated Backpacks (+ Sophisticated Core) eventually move from Forge to Neoforge, or support Neoforge in addition to Forge?

**Comment by P3pp3rF1y** [comment]
It supports it already in 1.20.1 (but admittedly there's very little difference there between forge and neo) and I certainly want to convert over to neoforge with the port to the next version (probably 1.20.4)

**Reply by championhestu** [reply]
Yeah, I'm aware they are compatible in 1.20, but I was asking for 1.21+ onwards. Many thanks for the response, and I'm glad I get to take Sophisticated Backpacks with me when I make the move!
